tests/fixtures/model_form_exclude.py

```diff
@@ -0,0 +1,23 @@
+class User(models.ModelForm):
+    name = models.CharField(max_length=255)
+    email = models.EmailField()
+
+    class Meta:
+        model = User
+        exclude = ('name',)
+
+class User2(ModelForm):
+    name = models.CharField(max_length=255)
+    email = models.EmailField()
+
+    class Meta:
+        model = User2
+        exclude = ('name',)
+
+class User3(ModelForm):
```

Comment:
I am not so familiarized with django forms, but do you know if is there a way to inherit from something like a `BaseForm`? This could be a test case.

Reply:
@rodolfolottin actually this checks is only valid for ModelForm.

This checker captures when people set `fields = '__all__'` which tells the form to use all fields from the model class, this could bring trouble when adding a new field into the model, since it could automatically add a field to the form that you don't want. Is better to explicitly set `fields` as a tuple listing each field you want on the form.

Reply:
Got it. But, acctually, my point was: is there a way to someone inherit from a `BaseForm`? I did not work with django model forms yet. But it would be something like:

```python
BaseModelForm(ModelForm):
    model = User

NewForm(BaseModelForm):
    fields = '__all__'
```

I think that this is a good test case to have written down.

Reply:
@rodolfolottin oh, right. Yes, this is a problem that we are not covering on this check since it would be more complex, it would involve checking all the classes defined on the module and also checking multiple multiple modules (which I don't know how to do). What do you think of opening a new issue for this? And if you want, you could investigate how to do it.